test/src/main/java/org/apache/accumulo/test/zookeeper/ZooCacheIT.java

```diff
@@ -228,4 +238,105 @@ public void testGetChildren() throws Exception {
           && zooCache.childrenCached(base + "/test3") == false;
     });
   }
+
+  @Test
+  public void testZookeeperRestart() throws Exception {
+    final String root = Constants.ZROOT + UUID.randomUUID();
+    final String base = root + Constants.ZTSERVERS;
+    TestZooCache zooCache = new TestZooCache(zk, Set.of(base));
```

Comment:
```suggestion
    final String root = Constants.ZROOT + UUID.randomUUID();
    final String base = root + Constants.ZTSERVERS;
    final String child = "test2";
    final String fullPath = base + "/" + child;
    final byte[] data1 = new byte[] {1, 2, 3, 4};
    final byte[] data2 = new byte[] {4, 3, 2, 1};
    TestZooCache zooCache = new TestZooCache(zk, Set.of(base));
```
Could add any of these

Reply:
changed data1 and data2 to strings in a later commit, wanted to detect if the code under test modified the bye array